pylot/planning/frenet_optimal_trajectory/FrenetOptimalTrajectory/src/FrenetOptimalTrajectory/FrenetOptimalTrajectory.h

```diff
@@ -0,0 +1,44 @@
+// Author: Edward Fang
+// Email: edward.fang@berkeley.edu
+//
+// This code is adapted from https://github.com/AtsushiSakai/PythonRobotics/tree/
+// master/PathPlanning/FrenetOptimalTrajectory.
+// Its author is Atsushi Sakai.
+//
+// Reference Papers:
+// - [Optimal Trajectory Generation for Dynamic Street Scenarios in a Frenet Frame]
+// (https://www.researchgate.net/profile/Moritz_Werling/publication/224156269_Optimal_Trajectory_Generation_for_Dynamic_Street_Scenarios_in_a_Frenet_Frame/links/54f749df0cf210398e9277af.pdf)
+// - [Optimal trajectory generation for dynamic street scenarios in a Frenet Frame]
+// (https://www.youtube.com/watch?v=Cj6tAQe7UCY)
+
+#ifndef FRENETOPTIMALTRAJECTORY_FRENETOPTIMALTRAJECTORY_H
+#define FRENETOPTIMALTRAJECTORY_FRENETOPTIMALTRAJECTORY_H
+
+#include <vector>
+#include "../CubicSpline/CubicSpline2D.h"
+#include "FrenetPath.h"
+
+// The FrenetOptimalTrajectory class.
+// See the references above for details.
```

Comment:
Please remove. These comments are not really useful.

Reply:
:+1: